TomTom SDK Examples/Examples/UseCase/Offline/Utils/OfflineMapHelper.swift

```diff
@@ -0,0 +1,137 @@
+//  © 2023 TomTom NV. All rights reserved.
+//
+//  This software is the proprietary copyright of TomTom NV and its subsidiaries and may be
+//  used for internal evaluation purposes or commercial use strictly subject to separate
+//  license agreement between you and TomTom NV. If you are the licensee, you are only permitted
+//  to use this software in accordance with the terms of your license agreement. If you are
+//  not the licensee, you are not authorized to use this software in any manner and should
+//  immediately return or destroy it.
+
+import Foundation
+
+// MARK: Offline Map Errors
+
+enum OfflineMapError: LocalizedError {
```

Comment:
could be put inside `OfflineMapHelper` since it's used only there.

Reply:
Done!